### ISSUE TITLE
[tableio] test_subevents is broken (Trac #1256)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1250]
`test_subvents.py` is not listed under `i3_test_scripts` but it is definitely broken. I think it should be removed or fixed.

Migrated from https://code.icecube.wisc.edu/ticket/1256
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "`test_subvents.py` is not listed under `i3_test_scripts` but it is definitely broken. I think it should be removed or fixed.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335639558230", 
    "component": "combo core", 
    "summary": "[tableio] test_subevents is broken", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T19:48:05", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T17:53:37: olivas changed milestone from "offline V15-09-00" to "offline V15-08-00"

**Comment by icecube-trac** [comment]
Trac update at 20150826T09:51:29: 
* jvansanten commented:

> In changeset 136877:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136877"
> Actually run test scripts. Fixes #1250
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas commented:

> Milestone offline V15-08-00 deleted


* olivas changed milestone from "offline V15-08-00" to ""
